pkgs/applications/virtualization/rust-hypervisor-firmware/default.nix

```diff
@@ -24,7 +25,9 @@ let
     };
   };
 
-  inherit (cross) rustPlatform;
+  rustPlatform = cross.makeRustPlatform {
```

Comment:
```suggestion
  # inherit (cross) rustPlatform;
  # ^ breaks because we are doing a no_std embedded build with a custom sysroot,
  # but the fast_cross rustc wrapper already passes a sysroot argument
  rustPlatform = cross.makeRustPlatform {
```